homeassistant/components/humidifier/device_action.py

```diff
@@ -6,6 +6,7 @@
 from homeassistant.components.device_automation import toggle_entity
 from homeassistant.const import (
     ATTR_ENTITY_ID,
+    ATTR_MODE,
```

Comment:
It can now be removed from `humidifier/const.py`, right?
https://github.com/home-assistant/core/blob/23de444fab0b0967edaa46d05fe0c35e16268c74/homeassistant/components/humidifier/const.py#L2

Reply:
Apart from that it looks good to me.

Reply:
I had issues with google assistant tests, but it should be fixed now.